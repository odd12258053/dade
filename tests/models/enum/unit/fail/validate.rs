use dade::{model, Result};
#[model]
enum TestModel {
    #[field(validate = validate_fn)]
    Value
}
fn main() {}
