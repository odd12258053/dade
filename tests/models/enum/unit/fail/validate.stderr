error: Only support alias term on the unit field.
 --> tests/models/enum/unit/fail/validate.rs:5:5
  |
5 |     Value
  |     ^^^^^

warning: unused import: `Result`
 --> tests/models/enum/unit/fail/validate.rs:1:19
  |
1 | use dade::{model, Result};
  |                   ^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
